src/components/spacings/spacings.visualroute.js

```diff
@@ -223,5 +223,35 @@ export const component = () => (
         <StackExample alignItems={prop} />
       </Spec>
     ))}
+    {[
+      'flex-start',
+      'flex-end',
+      'center',
+      'space-between',
+      'space-around',
+      'space-evenly',
+    ].map(prop => (
+      <Spec
+        key={`inline-justify-${prop}`}
+        label={`Inset - when justifyContent is ${prop}`}
```

Comment:
This is the VRT for inset not inline though 🤔

Reply:
oh the label is just wrong

Reply:
Sorry just a typo